scripts/newinstall.sh

```diff
@@ -755,6 +755,7 @@ n8l::generate_loader_bash() {
 		LSST_HOME="\$( cd "\$( dirname "\${BASH_SOURCE[0]}" )" && pwd )"
 
 		export EUPS_PATH="$eups_path"
+                export RUBIN_EUPS_PATH="\${EUPS_PATH}"
```

Comment:
The indenting is odd in all these lines.

Reply:
Tabs vs. spaces.  Fixing.